app/src/main/java/se/inera/intyg/minaintyg/logging/MonitoringLogServiceImpl.java

```diff
@@ -40,7 +46,8 @@ private String buildMessage(MonitoringEvent logEvent) {
   private enum MonitoringEvent {
     CITIZEN_LOGIN("Citizen '{}' logged in using login method '{}'"),
     CITIZEN_LOGOUT("Citizen '{}' logged out using login method '{}'"),
-    LIST_CERTIFICATES("Citizen '{}' listed '{}' certificates");
+    LIST_CERTIFICATES("Citizen '{}' listed '{}' certificates"),
+    GET_CERTIFICATE("Citizen '{}' retrieved certificate '{}'");
```

Comment:
Try and use the same logging as current minaintyg, when it exists. CERTIFICATE_READ("Certificate '{}' of type '{}' was read"),

Consider updating the log-method as well... calling it read...